Apps/WebClient/src/NewClientApp/src/views/errors/AppErrorView.vue

```diff
@@ -0,0 +1,49 @@
+<script setup lang="ts">
+import { computed } from "vue";
+
+import { AppErrorType } from "@/constants/errorType";
+import { useAppStore } from "@/stores/app";
+
+const appStore = useAppStore();
+
+const isTooManyRequests = computed<boolean>(() => {
+    return appStore.appError === AppErrorType.TooManyRequests;
+});
+</script>
+
+<template>
+    <v-layout>
```

Comment:
```suggestion
    <v-app>
```
Swapping to `<v-app>` matches App.vue & enables use of the fill-height class.